### PR TITLE
Add SSH key set up

### PR DESCRIPTION
**Pull request by chuanjin**
Signed-off-by: Chuan Jin <chuan.jin.813@gmail.com>

### CODE REVIEW

File: docs/articles/workflow/git-workflow.rst [hunk 1]
```diff
@@ -24,6 +24,22 @@ developers, as it is the repository the developer will typically push to.  On th
...
+    # This will generate a new ssh key for you
+
+    $ sudo apt-get install xclip
+    # Downloads and installs xclip. If you don't have `apt-get`, you might need to use another installer (like `yum`)
+
+    $ xclip -sel clip < ~/.ssh/id_rsa.pub
+    # Copies the contents of the id_rsa.pub file to your clipboard
+
+Click your avatar at GitHub page and go to Settings -> SSH and GPG keys, paste your key there.
+A long guide could be found here: https://help.github.com/articles/connecting-to-github-with-ssh/
```
Comment (tobsan) [inline docs/articles/workflow/git-workflow.rst]: This is too github specific, can we make it more abstract by saying something like "then add it to the account on the git web frontend" or something like that?

**Review by tobsan — CHANGES_REQUESTED** [review]
Just had a comment about github. Looks good otherwise.

File: docs/articles/workflow/git-workflow.rst (hunk 1 above)
Comment (chuanjin) [reply docs/articles/workflow/git-workflow.rst]: Sure, sounds good.

**Comment by chuanjin** [comment]
It should be more generic now. :)

File: docs/articles/workflow/git-workflow.rst [hunk 2]
```diff
@@ -24,6 +24,22 @@ developers, as it is the repository the developer will typically push to.  On th
...
+    # This will generate a new ssh key for you
+
+    $ sudo apt-get install xclip
+    # Downloads and installs xclip. If you don't have `apt-get`, you might need to use another installer (like `yum`)
+
+    $ xclip -sel clip < ~/.ssh/id_rsa.pub
+    # Copies the contents of the id_rsa.pub file to your clipboard
+
+Then add your new SSH key to the account on the git web frontend.
+A long guide could be found here: https://help.github.com/articles/connecting-to-github-with-ssh/
```
Comment (tobsan) [inline docs/articles/workflow/git-workflow.rst]: Sorry for being picky, but can we say "A long guide (for GitHub) can be found here" instead?

**Review by tobsan — CHANGES_REQUESTED** [review]

**Comment by tobsan** [comment]
@chuanjin did you see my new review?

File: docs/articles/workflow/git-workflow.rst (hunk 2 above)
Comment (chuanjin) [reply docs/articles/workflow/git-workflow.rst]: Sure! No problem.

**Review by tobsan — APPROVED** [review]

**Pull request merged**